src/features/workspaces/README.md

```
@@ -41,7 +41,18 @@ permissionsFor(ws.id)           // → WorkspacePermissions record
 hasPermission(ws.id, 'delete')  // → boolean
 ```
 
-**Current state:** wired in `ManagedWorkspaceSelector` (via `requiredPermission` prop — pass a `WorkspaceRelation` to disable non-permitted workspaces in the tree). Not yet broadly applied across the workspace detail and list UIs — that propagation is the next step. When in doubt, follow the selector as the reference implementation.
+**Propagation (PR1 — Kessel Permission Guards):** Permissions are now checked across all workspace surfaces:
+- **List table:** name link gated on `view`, row actions use correct per-action relations (`rename` for edit, `move` for move, `delete` for delete, `create` for sub-workspace)
+- **Create workspace:** parent selector passes `requiredPermission="create"` to `ManagedWorkspaceSelector`
+- **Detail page:** renders `<UnauthorizedAccess>` when `view` is denied; `WorkspaceActions` menu items disabled per relation
+- **Edit modal:** redirects if `rename` permission is denied (direct URL guard)
+
+**Role binding permission mapping (MVP):**
+| Operation | MVP Kessel relation | Post-MVP relation |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)